webapp/src/simulator.ts

```diff
@@ -33,6 +33,7 @@ export function setTranslations(translations: pxt.Map<string>) {
 }
 
 export function init(root: HTMLElement, cfg: SimulatorConfig) {
+    if (!root) return;
```

Comment:
any reason why we need this?

Reply:
Yes, boardview might not exist. 
It was throwing in cases where checkSession throws (due to an invalid session).